pkg/controller/controller.go

```diff
@@ -142,9 +142,8 @@ func (c *Controller) Run() error {
 			switch event.Type {
 			case "ADDED":
 				stopC := make(chan struct{})
-				nc, err := cluster.New(c.makeClusterConfig(), event.Object, stopC, &c.waitCluster)
-				if err != nil {
-					c.logger.Errorf("cluster (%q) is dead: %v", clusterName, err)
+				nc := cluster.New(c.makeClusterConfig(), event.Object, stopC, &c.waitCluster)
```

Comment:
error isn't very useful outside.
We already log it and report status inside New()